config/initializers/simple_form.rb

```diff
@@ -7,6 +7,9 @@
   require Rails.root.join(
     'lib', 'extensions', 'simple_form', 'components', 'html5_no_aria_required'
   )
+  require Rails.root.join(
+    'lib', 'extensions', 'simple_form', 'active_view_extensions', 'form_helper'
+  )
```

Comment:
this is really silly syntax IMO, by the time `initializers` is loaded, `lib` is in the load path, we can make these clearer to search for:
```suggestion
  require 'extensions/simple_form/components/html5_no_aria_required'
  require 'extensions/simple_form/active_view_extensions/form_helper'
```

Reply:
Yeah that reads a lot nicer! I was actually wondering if it could make sense to do something like what we have in `rails_helper.rb` loading all files in a folder, but wasn't too sure about doing that in runtime code.

https://github.com/18F/identity-idp/blob/37afb7efb468486efe3f7ed50ad7600f6329fc37/spec/rails_helper.rb#L25-L28

Reply:
Updated in 4b9c58f

Reply:
it would work fine in that case because `Dir[]` is what's doing the expansion of the `**/*.rb`

and realistically we have no plans to run Login.gov on windows where the path separator is backards